web3/_utils/transactions.py

```diff
@@ -195,7 +195,7 @@ def prepare_replacement_transaction(
             raise ValueError('Supplied gas price must exceed existing transaction gas price')
     else:
         generated_gas_price = web3.eth.generateGasPrice(new_transaction)
-        minimum_gas_price = int(math.ceil(current_transaction['gasPrice'] * 1.1))
```

Comment:
I'm thinking that the number here should be part of the function signature with `1.125` as the default value.